examples/test_common.h

```diff
@@ -0,0 +1,18 @@
+#ifndef test_common_h_
+#define test_common_h_
```

Comment:
Syntax is `XYZ_HDR_H`, that would in you case `TEST_COMMON_HDR_H`

Reply:
ok, see b68fad88